.github/workflows/migrations.yml

```diff
@@ -1,18 +1,50 @@
 name: migrations
-on:
-  pull_request:
-    paths:
-      - 'src/sentry/migrations/*'
+on: pull_request
 
 jobs:
+  did-migration-update:
+    name: check if migration changes
+    runs-on: ubuntu-20.04
+    timeout-minutes: 3
+    steps:
+      - name: Checkout sentry
+        uses: actions/checkout@v2
+
+      - name: Match migration files
+        uses: getsentry/paths-filter@v2
+        id: changes
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
+
+      - name: If no migration changes -> fail
+        shell: bash
+        if: needs.changes.outputs.migrations_added != 'true' || needs.changes.outputs.migrations_modified != 'true'
+        run: |
+          echo "Nothing for you to do. We always fail unless we want to trigger the dependent workflows below"
+          exit 1
```

Comment:
@billyvg I don't think this is what you had in mind. Is there another way to prevent the downstream workflows to trigger?
<img width="251" alt="image" src="https://user-images.githubusercontent.com/44410/109350802-25abec00-7846-11eb-9da2-1f12fcc0a91c.png">

Reply:
@armenzg Similar to outputs for a step, [jobs also have `outputs`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idoutputs) - we can then access them in dependent jobs with `needs.jobname.outputs`

Separate thing to note, `paths-filter` [provides a list of files](https://github.com/dorny/paths-filter#outputs) that was matched as an output